test/rules/first.spec.ts

```diff
@@ -77,12 +77,23 @@ ruleTester.run('first', rule, {
               import { x } from './foo';\
               import { z } from './baz';",
       errors: 3,
-      output:
+      output: [
         "import { y } from './bar';\
               var a = 1;\
               if (true) { x() };\
               import { x } from './foo';\
               import { z } from './baz';",
+        "import { y } from './bar';\
+              import { x } from './foo';\
+              var a = 1;\
+              if (true) { x() };\
+              import { z } from './baz';",
+        "import { y } from './bar';\
+              import { x } from './foo';\
+              import { z } from './baz';\
+              var a = 1;\
+              if (true) { x() };",
+      ],
```

Comment:
ESLint now auto fixes until the file stabilizes and requires all those transforms

Reply:
I couldn't find related docs in ESLint changelogs. Would you mind pointing out this for me?

Reply:
I picked this up from the test error messages:

```sh
  ● first › invalid › var a = 1;              import { y } from './bar';              if (true) { x() };              import { x } from './foo';              import { z } from './baz';

    assert.deepStrictEqual(received, expected)

    Expected value to deeply and strictly equal to:
      ["import { y } from './bar';              var a = 1;              if (true) { x() };              import { x } from './foo';              import { z } from './baz';"]
    Received:
      ["import { y } from './bar';              var a = 1;              if (true) { x() };              import { x } from './foo';              import { z } from './baz';", "import { y } from './bar';              import { x } from './foo';              var a = 1;              if (true) { x() };              import { z } from './baz';", "import { y } from './bar';              import { x } from './foo';              import { z } from './baz';              var a = 1;              if (true) { x() };"]
    
    Message:
      Multiple autofixes are required due to overlapping fix ranges - please use the array form of output to declare all of the expected autofix passes.
```